src/components/grouped-select/usa-state-population-data.ts

```diff
@@ -0,0 +1,206 @@
+export const stateData = [
```

Comment:
use from `generated-data/country-state.json`

Reply:
Removed this